terraform/Azure/only_private_Blob_storages/only_private_Blob_storages_test.rego

```diff
@@ -17,6 +17,6 @@ test_deny_blob_storage_with_non_private_acl {
 
 test_validate_deny_message {
   result:= deny with input as data.plan_mock.public_read_write_blob_storage
-  expected_deny_message:= "Deployment of not private blob storage bucket is not allowed"
+  expected_deny_message:= "Deployment of not private blobstorage bucket is not allowed"
```

Comment:
What's the message Product asked for? 'blobstorage' as one word?
if so.. we need to change the dir name, files names, and test methods as well, isn't it?